src/index.tsx

```diff
@@ -5,6 +5,11 @@ import '@fortawesome/fontawesome-free/css/all.css';
 import './styles/index.scss';
 
 import { App } from './App';
+import { AppTodoProvider } from './components/AppTodoContext/AppTodoContext';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
```

Comment:
It is recommended to use the `React.StrictMode` to wrap your application. This will help you to identify potential problems in your app during development. 🚀